exec/jdbc-all/pom.xml

```diff
@@ -562,7 +562,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>46600000</maxsize>
+                  <maxsize>50000000</maxsize>
```

Comment:
Unfortunately, I had to include them because there is an `OAuthUtils` class which uses OkHttp which is part of java-exec.  If you have some advice as to where else I could put that, I'm all ears.

The issue is that both the rest server and the http storage plugin needs to be able to access the same body of code for obtaining and refreshing tokens.

Reply:
@cgivre I _think_ you include in java-exec's pom and also explcitly _exclude_ in the java-exec dependency section in jdbc-all's pom.

Reply:
The problem is that it is needed for `java-exec`.  I tried this approach and unfortunately, it didn't work.